jwst/extract_1d/extract.py

```diff
@@ -614,21 +619,18 @@ def populate_time_keywords(input_model, output_model):
 
     log.debug("TSO data, so copying times from the INT_TIMES table.")
 
-    n = 0  # Counter for spectra in output_model.
-
-    for k in range(num_integ):  # for each spectrum or order
-        for _j in range(num_j):  # for each integration
+    for j in range(num_j):  # for each slit or order (EXTRACT1D extension)
+        spec = output_model.spec[j]
+        spec.time_scale = "UTC"
+        for k in range(len(spec.spec_table)):  # for each integration (table row)
             row = k + offset
-            spec = output_model.spec[n]  # n is incremented below
-            spec.int_num = int_num[row]
-            spec.time_scale = "UTC"
-            spec.start_time_mjd = start_time_mjd[row]
-            spec.mid_time_mjd = mid_time_mjd[row]
-            spec.end_time_mjd = end_time_mjd[row]
-            spec.start_tdb = start_tdb[row]
-            spec.mid_tdb = mid_tdb[row]
-            spec.end_tdb = end_tdb[row]
-            n += 1
+            spec.spec_table["INT_NUM"][k] = int_num[row]
```

Comment:
I guess these changed lines are what allow the stdatamodels PR to drop those attributes from the spec metadata?